server.py

```diff
@@ -19,6 +19,9 @@
 parser.add_argument('-k', '--key', type=str, dest='KEY', default=None, help='Secret key to authenticate clients')
 args = parser.parse_args()
 
+def index_with_fallback(obj, fst, snd):
```

Comment:
How do you feel about specifying alternative keys as command line arguments? E.g. `--key-overrides=html:quote;text:note` or something like that?

Otherwise this looks good to me!

Reply:
Nice- done.